src/app/chart/components/pokemon-chart/pokemon-chart.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ChartDataSets, ChartOptions, ChartType } from 'chart.js';
+import { Label } from 'ng2-charts';
+
+@Component({
+  selector: 'app-pokemon-chart',
+  templateUrl: './pokemon-chart.component.html',
+  styleUrls: ['./pokemon-chart.component.css']
+})
+export class PokemonChartComponent implements OnInit {
+  @Input() stats: Array<Array<Object>>;
+
+  static _barChartOptions: ChartOptions;
+  barChartLabels: Label[] = ['hp', 'attack', 'defense', 'special-attack', 'special-defense', 'speed'];
+  barChartType: ChartType = 'bar';
+  barChartLegend = false;
+  barChartPlugins = [];
+  barChartData: ChartDataSets[]
+  colors = ['#167a69', '#26b19c'];
+
+  constructor() {
+    PokemonChartComponent._barChartOptions = {
```

Comment:
as this is a static attribute it must be set in the class declaration, not in the constructor